### ISSUE TITLE
Extension View's publisher have greyscale rendering

### ISSUE THREAD

**Issue by octref** [microsoft/vscode#86192]
Ref #85143 

Version: 1.41.0-insider
Commit: 97855786a014be2440751b038b373c3726e11fe8
Date: 2019-12-03T05:31:49.954Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Linux x64 4.15.0-55-generic

![image](https://user-images.githubusercontent.com/4033249/70095208-07baf180-15d9-11ea-8efe-ba906c61b1c1.png)

**Comment by Tyriar** [comment]
The title of disabled items is also greyscale:

![image](https://user-images.githubusercontent.com/2193314/70192469-a403f780-16b0-11ea-888a-0149be287981.png)

**Comment by bpasero** [comment]
If you know the background color precisely you can compute the opacity by blending colors together and thus avoid opacity which is likely causing the issue in this case. 

Example: https://github.com/microsoft/vscode/blob/c4a0949f2c66d5179126c9211319ba5df6a95f38/src/vs/workbench/browser/parts/titlebar/titlebarPart.ts#L510

**Comment by joaomoreno** [comment]
> I am assuming the list / tree widget takes care of this grey scale rendering and the elements inside it does not need to, is not it @joaomoreno ?

Not sure I understood the question. The tree/list doesn't prevent you from having subpixel AA. But it also doesn't guarantee it for its users, since there are things done on row content, eg opacity, which could prevent it.

**Comment by sandy081** [comment]
Used the trick @bpasero suggested and it worked

![image](https://user-images.githubusercontent.com/10746682/84487076-87b7c780-ac9e-11ea-9e92-7f7732531746.png)

**Comment by sandy081** [comment]
Here are the layers rendered (which I got with the help of @joaomoreno)

![image](https://user-images.githubusercontent.com/10746682/84520508-f9f6cf00-acd3-11ea-834e-cf73b6afd63c.png)

As from @joaomoreno 
It seems there are no new layers around publisher field it could be another bug altogether then and could even potentially be an electron issue

@bpasero @deepak1556 any idea if this is an electron issue and if not shall we have to follow this blending trick at all such places?

**Comment by bpasero** [comment]
@sandy081 we have had a lot of discussion with the Chrome team in the past and I think Alex made a good list of conditions under which Chrome may decide to switch to greyscale rendering (https://bugs.chromium.org/p/chromium/issues/detail?id=1016062#c31). I doubt it is Electron to blame, rather Chrome, but then again unlikely to get a fix imho.

As for `opacity` in our code: this is not a guarantee for loosing LCD rendering. A lot of places use `opacity` and render just fine. However, in some specific cases we seem to be loosing LCD rendering and you can find all issues we filed via this query: https://github.com/microsoft/vscode/labels/lcd-text-rendering

As you can see, some of these issues are still open or closed "as designed", simply because the technical solution was not straight forward. 

tl;dr;: we try our best to get LCD rendering, but by no means are we thinking that we get to 100%. Just to give an example: notification toasts do not render with LCD rendering because of many factors, such as animations, border shadow, etc. I did not have the energy to make it work there.

Ah, and btw other browsers are fine, Firefox always renders in LCD, no matter what. And this is even platform specific, e.g. I was not able to see this issue on Windows, only Linux.

**Comment by rzhao271** [comment]
@bpasero @sandy081 is there an easy way to verify this issue? It seems there's a lot going on here

**Comment by sandy081** [comment]
This can be verified only on Linux.
- Open extensions view and zoom in and make sure publisher's name is not rendered with grey scale.

**Comment by rzhao271** [comment]
I had a bit of trouble with this because I realized the best way is to take a screenshot and then zoom in, not the other way around. Verified!
